src/Donors/resources/admin/components/DonorDetailsPage/Tabs/Profile/CustomFields/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { __ } from "@wordpress/i18n";
+import { useFormContext } from "react-hook-form";
+import AdminSection, { AdminSectionField } from '@givewp/components/AdminDetailsPage/AdminSection';
+import BlankSlate from './BlankSlate';
+import styles from './styles.module.scss';
+
+interface CustomField {
+    label: string;
+    value: string;
+}
+
+export default function CustomFields() {
+    const { watch } = useFormContext();
+    const customFields: CustomField[] = watch('customFields') || [];
```

Comment:
Do we need to be watching fields if just displaying them?  

As a side note, react-hook-form has a hook for better performance called [useWatch](https://react-hook-form.com/docs/usewatch)